### ISSUE TITLE
Add Timezone 

### ISSUE THREAD

**Issue by dragethy** [cmfcmf/OpenWeatherMap-PHP-Api#73]
Please Add timezone for Sunrise and Sunset.

The DateTimezone for the OWM Api is GMT but for the right Sunrise and Sunset Datetime we need the correct Timezone.

**Comment by cmfcmf** [comment]
Hi @dragethy, 

I think I fixed all timezone related errors in #75. Let me know if it works for you :100:

**Comment by dragethy** [comment]
Your Solution works

`This is your Output: 04:32 (UTC)`

a better solution is to calculate the Time for the right Timezone for the place
my verry hardcoded Solution ist for Germany
```
$date_sunset =  $forecast->sun->set ;

$date_sunset->modify( '+2 hours' );

$date_sunrise = $forecast->sun->rise;

$date_sunrise->modify('+2 hours');
```

**Comment by cmfcmf** [comment]
You should rather do
```php
$forecast->sun->set->setTimezone(new \DateTimezone('Europe/Berlin'))->format('d.m.Y H:i (e)')
```

**Comment by dragethy** [comment]
Great works like a charm

**Comment by cmfcmf** [comment]
> Great works like a charm

Ok, great! Released version 2.1.1.
